src/shared/consts.tsx

```diff
@@ -312,11 +312,12 @@ export const CONDITION_TYPES_TO_HAVE_RANGE_VALUE = [
   ConditionType.OutsideOfTimes,
 ];
 
-export const GENERAL_REPORT_NAME = 'report';
+export const LEGACY_GENERAL_REPORT_NAME = 'report';
+export const NEW_GENERAL_REPORT_NAME = 'responses';
```

Comment:
I missed this one. It's another small suggestion, but similar to my previous suggestion of continuing to use  using `reportHeader` instead of `newReportHeader`, I think it's ok to just use `GENERAL_REPORT_NAME`. Then during flag cleanup, only the removal of `LEGACY_GENERAL_REPORT_NAME` is needed.